prime-router/src/testIntegration/kotlin/datatests/mappinginventory/catchall/orcobr/OBRToSpecimenTests.kt

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.prime.router.datatests.mappinginventory.orcobr
+
+import gov.cdc.prime.router.datatests.mappinginventory.verifyHL7ToFHIRToHL7Mapping
+import org.junit.jupiter.api.Test
+
+class OBRToSpecimenTests {
```

Comment:
I'm open to different names/locations for these tests. They made sense at the time. These tests specifically test that
- FHIR->HL7 OBR/Specimen mappings work despite being linked in the FHIR through different paths in ORU/OML.
- HL7->FHIR->HL7 OBR/Specimen mappings work for ORM despite not having the Specimen resource.